### DISCUSSION TITLE
Why does DIGIT supports only POST requests?

### DISCUSSION THREAD

**Discussion by atulgupta2024** [egovernments/Digit-Core#367] [DIGIT Platform]
While working on PUCAR, I had designed a few GET APIS, only to realize that the authentication token is being passed in the body of the post and not the header. Passing authorization tokes in request header is a standard technique. 

Any reason why we don't support that and rather pass the tokens in the request body?

**Comment by subhashini-egov** [comment]
@atulgupta2024 here is a[ document](https://docs.google.com/document/d/18Mp7n_PqQXOdPiPgd5beGa5ovOnuABPJ/edit?usp=sharing&ouid=100909912909872122080&rtpof=true&sd=true) that was shared by some of the earliest architects of DIGIT that explains the reasoning. Largely, it was to stay protocol independent.
